Minimum-viable print

andrewmaier opened the pull request:
Adds a link in the footer:
![image](https://user-images.githubusercontent.com/2970/33737679-0a3b3f7e-db65-11e7-81dc-f645afd16d85.png)

which opens up a two-column print layout with every page in the guide.

tbaxter-18f commented on _includes/site-footer.html:
```diff
@@ -5,13 +5,14 @@
     </span>
     <nav>
       <a href="{{ site.baseurl}}/nav-tips/" class="button button-nav-tips">Navigation tips</a>
+      <a href="{{ site.baseurl}}/print/" class="button button-print" target="_blank">Print this guide</a>
```
Why the use of `_blank`?

andrewmaier replied:
BC I didn't show a header/footer. I could add them back in and remove the `_blank` and hide them via the `print.css`. I just wanted a way for people to get back to the guide and figured `_blank` solved it :)

tbaxter-18f replied:
From the research I've seen -- which is far from definitive -- generally speaking you'd want to leave opening new tabs and navigating back in the hands of the users, reserving `_blank` for cases in which you've got a big workflow that could be lost, their place might be lost, or leaving and coming back could otherwise be destructive. That said, I think the research is a bit thin, and I definitely see your case for using it here.

andrewmaier replied:
I harbor no strong preferences on `_blank`. I literally just talked myself through how I could design it better in response to your initial comment!

tbaxter-18f commented on _pages/print.html:
```diff
@@ -0,0 +1,23 @@
+  <body class="layout-print">
+    {% for page in site.pages %}
+      {% unless page.title == "Introduction" %}
+        <div class="article">
```
There's probably an underlying reason I'm not aware of, but why `.article` rather than an `article` element?

tbaxter-18f commented on stylesheets/screen.css:
```diff
@@ -435,12 +437,17 @@ input[type="search"]:focus::-webkit-search-cancel-button::after{
+.site-footer .button.button-print,
 .site-footer .button.button-issue{
-	border-radius: 0 2px 2px 0;
+	border-radius: 0 2px 2px 0 !important;
```
Why is the `important` needed?

andrewmaier replied:
It wasn't! Fixed.

andrewmaier commented:
@tbaxter-18f Rebased this branch and fixed some things based on your feedback! The print page uses `article` elements, no longer opens in `_blank`, and I removed the `!important` bits from `screen.css`.

wslack commented:
cc @line47 if there's anything useful with this template wise

andrewmaier commented:
@line47 I'm not super proud of this, but here's [the template I wrote](https://github.com/18F/content-guide/blob/e8474c4764b11d498e30726dc87b96e5216ca82a/_pages/print.html) to make the print page follow the programmatically generated table-of-contents.
